package.json

```diff
@@ -4,7 +4,8 @@
     "start": "npm run clean && NODE_ENV=development webpack --watch",
     "build": "npm run clean && NODE_ENV=production webpack",
     "build:dev": "npm run clean && NODE_ENV=development webpack",
-    "clean": "rm -rf public/dist"
+    "clean": "rm -rf public/dist",
+    "test": "jest"
   },
   "babel": {
     "presets": [
```

Comment:
@DFurnes should I also add `flow` in the presets similar to [Phoenix Next](https://github.com/DoSomething/phoenix-next/blob/dev/package.json#L22)? 

Also, I think I can skip adding 
```
{
  "presets": ["es2015", "react"]
}
``` 
like it says [here (under Additional Configuration](https://facebook.github.io/jest/docs/en/getting-started.html#content)?

Reply:
We're not using Flow in Rogue at the moment, so nope! Safe to leave out. And yep, can leave the Babel configuration as is. 😄

Reply:
awesome, thanks @DFurnes !